ng2-components/ng2-alfresco-documentlist/README.md

```diff
@@ -437,18 +452,53 @@ For a full list of available `format` values please refer to [DatePipe](https://
 
 It is possible providing custom column/cell template that may contain other Angular components or HTML elmements:
 
+Every cell in the DataTable component is bound to the dynamic data context containing the following properties:
+
+| Name | Type | Description |
+| --- | --- | --- |
+| data | [DataTableAdapter](https://github.com/Alfresco/alfresco-ng2-components/tree/master/ng2-components/ng2-alfresco-datatable#data-sources) | Data adapter instance. |
+| row | [DataRow](https://github.com/Alfresco/alfresco-ng2-components/tree/master/ng2-components/ng2-alfresco-datatable#data-sources) | Current data row instance.  |
+| col | [DataColumn](https://github.com/Alfresco/alfresco-ng2-components/tree/master/ng2-components/ng2-alfresco-datatable#data-sources) | Current data column instance. |
+
+You can use all three properties to gain full access to underlying data from within your custom templates. 
+In order to wire HTML templates with the data context you will need defining a variable that is bound to `$implicit` like shown below:
+
 ```html
-<content-column
-        title="{{'DOCUMENT_LIST.COLUMNS.DISPLAY_NAME' | translate}}"
-        key="name"
-        sortable="true"
-        class="full-width ellipsis-cell">
+<template let-context="$implicit">
+    <!-- template body -->
+</template>
+```
+
+The format of naming is `let-VARIABLE_NAME="$implicit"` where `VARIABLE_NAME` is the name of the variable you want binding template data context to.
+
+Getting a cell value from the underlying DataTableAdapter:
+
+```ts
+context.data.getValue(entry.row, entry.col);
```

Comment:
which one is correct?
- context.data.getValue(entry.row, entry.col);
- entry.data.getValue(entry.row, entry.col);